src/main/java/mike/TaskList.java

```diff
@@ -0,0 +1,83 @@
+package mike;
+
+import mike.task.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A List of Tasks
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
+
+    /**
+     * Creates a taskList from a list of Task Objects
+     * @param tasks
+     */
+    public TaskList(List<Task> tasks) {
+        this.taskList = new ArrayList();
+        this.taskList.addAll(tasks);
+    }
+
+    /**
+     * Creates an empty taskList
+     */
+    public TaskList() {
+        this.taskList = new ArrayList();
+    }
+
+    /**
+     * Add a task to the taskList
+     *
+     * @param task the task to be added to the taskList
+     */
+    public void add(Task task) {
```

Comment:
I agree with @yeoutzer. The function names use verbs which not only follows the quality guideline but makes it easy to understand purpose of each method.